src/WebJobs.Script.WebHost/Controllers/KeysController.cs

```diff
@@ -38,6 +38,7 @@ public KeysController(IOptions<ScriptApplicationHostOptions> applicationOptions,
         }
 
         [HttpGet]
+        [HttpPost]
```

Comment:
Need to add test. There's already a KeysControllerTests suite. Just add a POST test matching the existing GET test

Reply:
I see now that those tests are direct method tests - not full http tests.